pre_commit_hooks/pretty_format_json.py

```diff
@@ -24,6 +24,25 @@ def _autofix(filename, new_contents):
         f.write(new_contents)
 
 
+def parse_indent(s):
+    # type: (str) -> str
+    try:
+        int_indentation_spec = int(s)
+        if int_indentation_spec >= 0:
+            return int_indentation_spec * ' '
+        else:
+            raise ValueError(
+                'Negative integer supplied to construct JSON indentation delimiter. ',
+            )
```

Comment:
is this exception ever actually propagating out? seems it's just going to get clobbered below (and you'll only see the message on python3, which also shows the original exception)